java/problems/permutationsOfString

```diff
@@ -0,0 +1,36 @@
+// Java program to print all the permutations
+// of the given string
+public class Permutation {
```

Comment:
Explain more about the problem, with some test cases and validate those test cases with your drive code.